### ISSUE TITLE
[Enhancement] pagerduty_team data source

### ISSUE THREAD

**Issue by paddie** [terraform-providers/terraform-provider-pagerduty#64]
Hi,

we've upgraded to `standard` and have found it to be much nicer to introduce terraform at the boundary of the teams. Meaning, we we want to create teams, schedules, users etc. by hand, but want to create services, service integrations etc. using terraform.

This is currently impossible without first performing an import of an existing resource. We'd much rather be able to just create new services and escalation policies using a data resource instead of a resource.. ehrm.. resource.

In short we'd like this:

```terraform
data "pagerduty_team" "team" {
    name = "${var.team_name}"
}
```

### Terraform Version
Run `terraform -v` to show the version. If you are not running the latest version of Terraform, please upgrade because your issue may have already been fixed.

Terraform v0.11.3
+ provider.aws v1.9.0
+ provider.pagerduty v0.1.3
